File: VietOCR.NET/tags/Release-0.8/Postprocessing/Processor.cs
namespace net.sourceforge.vietocr.postprocessing
{
    using System;

    /**
     *
     * @author Quan Nguyen (nguyenq@users.sf.net)
     */
    public class Processor
    {
        
        public static string PostProcess(string text, string langCode)
        {
            IPostProcessor processor = ProcessorFactory.createProcessor((ISO639) Enum.Parse(typeof(ISO639), langCode));
            return processor.PostProcess(text);
        }
    }
}

File: VietOCR/tags/Release-0.9.8/src/net/sourceforge/vietocr/postprocessing/Processor.java
/*
 * Processor.java
 *
 * Created on April 10, 2008, 5:58 PM
 */

package net.sourceforge.vietocr.postprocessing;

/**
 *
 * @author Quan Nguyen (nguyenq@users.sf.net)
 */
public class Processor {
    public static String postProcess(String text, String langCode)
    {
        IPostProcessor processor = ProcessorFactory.createProcessor(ISO639.valueOf(langCode));
        return processor.postProcess(text);
    }
}
